### DISCUSSION TITLE
Add a "correction" field on the user feedback info

### DISCUSSION THREAD

**Discussion by almirb** [langfuse/langfuse#1415] [Ideas]
### Describe the feature or potential improvement

When making considerations as an user feedback, would be nice to have a text field to propose a correct answer for the LLM.  This corrections could be use this "corrections" do add data to a RAG database.

**Comment by marcklingen** [comment]
that's a great idea. you could add this to the metadata field in the meantime.

**Comment by marcklingen** [comment]
More context on this from [Discord](https://discord.com/channels/1111061815649124414/1111063352593088633/1254901320461324338)
![CleanShot 2024-07-02 at 02 52 35](https://github.com/langfuse/langfuse/assets/2834609/45aaa2b1-0cc4-477b-a581-1ecdc85e275e)


cc @marliessophie

**Comment by jannikmaierhoefer** [comment]
Hi @almirb, this has been added: 
<img width="2186" height="1242" alt="CleanShot 2026-01-09 at 14 37 20@2x" src="https://github.com/user-attachments/assets/8529dab4-07d3-44ff-98db-b18fa59d518a" />

Let us know if you have any feedback!

**Reply by almirb** [reply]
> Hi @almirb, this has been added: <img alt="CleanShot 2026-01-09 at 14 37 20@2x" width="2000" height="1242" src="https://private-user-images.githubusercontent.com/48529566/533911214-8529dab4-07d3-44ff-98db-b18fa59d518a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njc5NjYyMjYsIm5iZiI6MTc2Nzk2NTkyNiwicGF0aCI6Ii80ODUyOTU2Ni81MzM5MTEyMTQtODUyOWRhYjQtMDdkMy00NGZmLTk4ZGItYjE4ZmE1OWQ1MThhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTA5VDEzMzg0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmMjc3YmQ2OTA5MTRkMDZmOWE2NTRjNjIwY2Q3NTI1ZjgyYjZhMmIxZmQ4MGUzMDEzZWU1ZDNkMTA5NjA4MGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NSCQmum4BjpS7P7H8dd6V52pl0rFkyG_oajqgOnpZzw">
> 
> Let us know if you have any feedback!

Thanks for this addition!
I'll test It ASAP!
